### DISCUSSION TITLE
Issue with previous global installation of typescript-language-server

### DISCUSSION THREAD

**Discussion by rushilmehra** [williamboman/nvim-lsp-installer#591] [Q&A] [answered]
I previously used `nvim-lsp` by installing `typescript-language-server` globally. I tried removing that global installation and using `nvim-lsp-installer`, but now i'm getting an issue: `Unable to find executable. Please check your path and ensure the server is installed`. I'm new to all of this, so any help is appreciated. Thanks

**Comment by williamboman** [comment, marked as answer]
Hey! nvim-lsp-installer doesn't install executables to a location that is commonly found in `PATH`. Did you see the setup section in the README https://github.com/williamboman/nvim-lsp-installer#setup?

**Reply by rushilmehra** [reply]
Lol I should have read more carefully. I think i'm overriding this installer's installations. Let me try to fix it
